src/Hng.Application/Features/Notifications/Commands/CreateNotificationCommand.cs

```diff
@@ -3,14 +3,14 @@
 
 namespace Hng.Application.Features.Notifications.Commands
 {
-    public class CreateNotificationCommand : IRequest<NotificationDto>
+    public class CreateNotificationCommand : IRequest<NotificationResult>
     {
-        public CreateNotificationCommand(CreateNotificationDto notificationDto, string loggedInUserId)
+        public CreateNotificationCommand(CreateNotificationDto createNotificationDto, string loggedInUserId)
```

Comment:
this would make you have to get logged in user from the controller and pass it into the command, use the GetLoggedInUserId method found in the authentication service class in hng.infrastructure.

Use it to get the logged in user in the command handler directly , instead of passing it from the controller

Reply:
Sorry for pushing my app settings, it escape my mind. Will work on the corrections.